### ISSUE TITLE
#오늘의레딧 https://www.reddit.com/r/MachineLearning/ 래딧은 읽을 거리가 다양합니다.

### ISSUE THREAD

**Issue by hochan222** [hochan222/Search#4]
1. 여기 있어요! "하지만 신경망은 무엇입니까?"- 심화 학습, 1 부
2. 감독되지 않은 이미지 - 이미지 번역 네트워크
3. 이미지 간 신경 전달
4. 예제 및 응용 프로그램을 사용한 단일 값 분해 방법
5. DeepMind의 Richard Sutton - AI의 장기 : 시간차 학습
6. Matlab 2017b에 대한 생각?
7. 해석 가능한 컨볼 루션 뉴럴 네트워크
8. 확률 적 최적화 기가 선형 변환입니까?
9. 좋은 컨퍼런스 목록과 마감일?
10. WaveNet이 Google 길잡이에서 실행됩니다.
11. 왜 batchnorm은 매개 변수를 가지고 있습니까?
12. 이 입문서와 함께 작은 벡터 튜토리얼을 통해 단어 벡터화에 대해 배웁니다.
13. MLP 또는 RBF로 정책을 매개 변수화하기위한 영향 / 차이점은 무엇입니까?
14. 데이터 과학과 소프트웨어 엔지니어링 사이에서 어떻게 GAP을 채우고 있습니까?
15. GPU 또는 CPU 용 Tensorflow를 설치해야합니까?
16. 정밀도 및 회상 : 개요
17. 지형 공간 데이터 신경망
18. Rasa Core : 대화식 소프트웨어를위한 기계 학습 기반 대화 엔진
19. 감정 이해 - Keras에서 pyTorch에 이르기까지
20. PyTorch에서 구현 된 신경 튜링 기계 (NTM)
1. https://youtu.be/aircAruvnKk
2. https://www.reddit.com/…/r_unsupervised_imagetoimage_trans…/
3. https://www.reddit.com/…/r_neural_color_transfer_between_i…/
4. https://blog.statsbot.co/singular-value-decomposition-tutor…
5. https://www.youtube.com/watch?v=EeMCEQa85tw
6. https://www.reddit.com/…/comments/74jsz4/d_thoughts_on_mat…/
7. https://arxiv.org/abs/1710.00935
8. https://www.reddit.com/…/d_are_stochastic_optimizers_linea…/
9. https://www.reddit.com/…/d_a_good_list_of_conferences_and_…/
10. https://deepmind.com/blog/wavenet-launches-google-assistant/
11. https://www.reddit.com/…/r_why_does_batchnorm_have_any_par…/
12. http://www.theodo.fr/blog/2017/09/day-words-became-vectors/
13. https://www.reddit.com/…/d_any_impactdifference_to_paramet…/
14. https://www.reddit.com/…/d_how_filling_the_gaps_between_da…/
15. https://www.reddit.com/…/d_should_i_install_tensorflow_for…/
16. https://bbabenko.github.io/prs/
17. https://www.reddit.com/…/74eb1h/d_geospatial_data_neural_n…/
18. https://github.com/RasaHQ/rasa_core
19. https://medium.com/…/understanding-emotions-from-keras-to-p…
20. https://github.com/loudinthecloud/pytorch-ntm
